src/goreleaser.ts

```diff
@@ -18,239 +16,315 @@ const configOption: Fig.Option = {
     template: "filepaths",
   },
 };
-
 const rmDistOption: Fig.Option = {
   name: "--rm-dist",
   description: "Removes the dist folder",
 };
-
 const parallelismOption: Fig.Option = {
   name: ["-p", "--parallelism"],
   description: "Amount tasks to run concurrently (default: number of CPUs)",
   args: {
     name: "parallelism",
   },
 };
-
 const skipValidateOption: Fig.Option = {
   name: "--skip-validate",
   description: "Skips several sanity checks",
 };
-
 const snapshotOption: Fig.Option = {
   name: "--snapshot",
   description:
     "Generate an unversioned snapshot build, skipping all validations",
 };
-
 const timeoutOption: Fig.Option = {
   name: "--timeout",
   description: "Timeout to the entire build process (default 30m0s)",
   args: {
     name: "duration",
   },
 };
-
 const completionSpec: Fig.Spec = {
   name: "goreleaser",
-  description: "GoReleaser is a release automation tool for Go projects",
+  description: "Deliver Go binaries as fast and easily as possible",
   subcommands: [
     {
-      name: "build",
+      name: ["b", "build"],
       description: "Builds the current project",
       options: [
-        debugOption,
-        configOption,
-        rmDistOption,
-        parallelismOption,
-        skipValidateOption,
-        snapshotOption,
-        timeoutOption,
+        {
+          name: ["--config", "-f"],
+          description: "Load configuration from file",
+          args: { name: "config" },
```

Comment:
Filepaths is likely missing in the goreleaser codebase:
```suggestion
          args: { name: "config", template: "filepaths" },
```